sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/AsyncAutoCloseable.java

```diff
@@ -8,18 +8,16 @@
 /**
  * Asynchronous class to close resources.
  */
-public interface AsyncAutoCloseable extends AutoCloseable {
+public interface AsyncAutoCloseable {
 
     /**
      * Begins the close operation. If one is in progress, will return that existing close operation.
      *
      * @return A mono representing the close operation.
      */
-    Mono<Void> closeAsync();
+    default Mono<Void> closeAsync() {
+        return closeAsync("");
+    }
 
-    /**
-     * Synchronous close operation. Blocks until the entire operation is complete.
-     */
-    @Override
-    void close();
+    Mono<Void> closeAsync(String closeMessage);
```

Comment:
I don't think we need this method for the interface. Implementing ones can use a close message if they want, but it isn't necessary.